app/src/main/java/io/github/droidkaigi/confsched/util/LocaleUtil.java

```diff
@@ -103,6 +103,20 @@ public static Date getDisplayDate(@NonNull Date date, Context context) {
         }
     }
 
+    public static Date getConfTimezoneCurrentDate() {
```

Comment:
OK, should we create DateFormat instance per each method call? Or make them constant.